examples/gallery/lucid_pipe_tee.py

```diff
@@ -66,8 +67,7 @@
 # . Read and display the geometry file.
 
 # The file contains several unmeshed parts as you would get after you imported from CAD.
-
-file_name = prime.examples.download_pipe_tee_fmd()
+file_name = prime.examples.get_file(Examples.PIPE_TEE_PMDAT)
```

Comment:
@AlejandroFernandezLuces 
```suggestion
file_name = prime.examples.get_file(Examples.PIPE_TEE_FMD)
```